doc/sequence_diagram.puml

```diff
@@ -1,6 +1,7 @@
 @startuml
 
-skinparam dpi 300
+' You can paste this content into https://plantuml-editor.kkeisuke.com/ to generate the PNG
+skinparam dpi 150
```

Comment:
NB, a DPI of 300 resulted in the image being cut-off on each of the 3 or 4 PlantUML-to-PNG services I tried. (The one linked in the comment was by far the fastest and most user friendly, even if it is someone's personal project).